src/zeit/content/video/ftesting.zcml

```diff
@@ -9,9 +9,7 @@
   <include package="zeit.content.video" />
   <include package="zeit.content.video.browser" />
 
-  <include package="zeit.content.author" />
-  <include package="zeit.content.author.browser" />
-
   <include package="zeit.solr" />
+  <include package="zeit.content.author" file="ftesting.zcml" />
```

Comment:
Warum ist das klarer? Auch in `zeit.content.article.ftesting` steht es ohne `ftesting.zcml`.